lib/data/tools/select.dart

```diff
@@ -30,6 +30,27 @@ class Select extends Tool {
     selectResult.pageIndex = -1;
   }
 
+  Color? getDominantStrokeColor(){
```

Comment:
For completeness we could add a `doneSelecting` check here. What do you think?
```suggestion
  Color? getDominantStrokeColor(){
    if (!doneSelecting) return null;

```

Reply:
Makes sense.
Didn't had the whole ``doneSelecting`` thing in mind.
Definitely gonna look twice for future pull requests.